lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, module, function=None):
 
         # If URLs need to be modified or added for specific purposes, use .update() on the url_catalog dictionary
         self.get_urls = {'organizations': '/organizations',
-                         'networks': '/organizations/{org_id}/networks',
+                         'network': '/organizations/{org_id}/networks',
```

Comment:
Not sure why this is singular now ?

Reply:
Consistency. The module is network. I set networks early on when I first wrote the module, before meraki_network was a thing.

Reply:
Ok, so you're standardizing on the module name, not the internal REST call.
I guess both are fine, as it's not exactly exposed to the user so it doesn't really matter.